lang/Io/Portfile

```diff
@@ -24,8 +24,9 @@ long_description        ${description}. The ideas in Io are mostly inspired \
 
 homepage                http://www.iolanguage.com/
 
-checksums               rmd160  c97575bc3e915ef3d68b396900418f77261adea6 \
-                        sha256  a5ca07b4d211995a0cd9442e1d752b2824444bee018c6db31899ddd6c5c87c4f
+checksums           rmd160  ebf5d206b115329456df169aaaad33f431a92bfc \
+                    sha256  febf9b6f4174e3d9fee179b5d41a81dba8b9660ece7a1cecee7452f5e9e460ea \
+                    size    9154022
```

Comment:
Although it isn't strictly required, it would be nice if this was indented at the same column as the rest.